### PR TITLE
Robots_hw

### PR DESCRIPTION
**Pull request by Lansaro**
Check this out

Robotzzzzzzzzz

### CODE REVIEW

File: LeoTishin-Lansaro/hw_4/car_lot.rb [hunk 1]
```diff
@@ -36,3 +36,4 @@ def add_cars(*all_cars)
 #APPROACH - B
 cars = add_cars(tesla, ford, porshe, deloraen, toyota)
 show_all_cars(cars)
+puts "The "
```
Comment (tgaff) [inline LeoTishin-Lansaro/hw_4/car_lot.rb]: hw_4 change?

Comment (Lansaro) [inline LeoTishin-Lansaro/hw_4/car_lot.rb]: Yeah, looks like it did go through in my terminal

File: LeoTishin-Lansaro/hw_6/rise_of_robot.rb [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+      "I can not fly {crying}"
+    end
+  end
+
+  def laser_fighting_skills
+    "I have laser fighting skills" unless type != "Android"
+  end
+end
+
+Robot.make_robots(50)
```
Comment (tgaff) [inline LeoTishin-Lansaro/hw_6/rise_of_robot.rb]: could you call some of the instance methods?  I'd like to see them in use.

File: LeoTishin-Lansaro/hw_6/rise_of_robot.rb [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+end
+
+Robot.make_robots(5)
+
+travis_bot = Robot.new("travis", "Android", "Chicago")
+leo_bot = Robot.new("leo", "Android", "Nairobi")
+
+travis_bot.empathy # =>
+leo_bot.flying_skills
+Robot.make_robots(4)
```
Comment (tgaff) [inline LeoTishin-Lansaro/hw_6/rise_of_robot.rb]: Great work Leo!

File: LeoTishin-Lansaro/hw_7/Thermostat_practice_l [hunk 4]
```diff
@@ -0,0 +1,32 @@
+require 'pry'
```
Comment (tgaff) [inline LeoTishin-Lansaro/hw_7/Thermostat_practice_l]: @lansaro please rename this to end in `.rb`

**Comment by tgaff** [comment]
looks great Leo!  If you can rename the one file, I'll merge it :+1:

File: LeoTishin-Lansaro/hw_7/Thermostat_practice_l (hunk 4 above)
Comment (tgaff) [inline LeoTishin-Lansaro/hw_7/Thermostat_practice_l]: actually this should be in the hw_6 directory also

**Comment by tgaff** [comment]
OK, now we're all good here!

**Pull request merged**